### PR TITLE
rg-65: Add Icon Input

### PR DESCRIPTION
**Pull request by Valentyn13**
Dependency:

- Add className prop to FormGroup component

- Add className prop to Input component

Edit IconInput component
Usage of IconInput component with FormGroup:
icon-input-usage

### CODE REVIEW

File: frontend/src/bundles/common/components/icon-input/icon-input.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    className = '',
+    width = '200px',
+    ...otherProperties
+}: Properties): JSX.Element => {
+    const { onChange } = otherProperties;
+
+    return (
+        <label className={styles.input__wrapper} style={{ width }}>
+            {children}
+            <input
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/common/components/icon-input/icon-input.tsx:23]: Try to expand the functionality of the existing input component

File: frontend/src/bundles/common/components/icon-input/icon-input.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                className={clsx(styles.icon_input, className)}
+                {...otherProperties}
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/common/components/icon-input/icon-input.tsx:25]: Move this line up

File: frontend/src/bundles/common/components/icon-input/icon-input.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import clsx from 'clsx';
+import { type InputHTMLAttributes, type ReactNode } from 'react';
+
+import styles from './styles.module.scss';
+
+interface Properties extends InputHTMLAttributes<HTMLInputElement> {
+    className?: string;
+    width?: string;
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/common/components/icon-input/icon-input.tsx:8]: Width can be set via `calssName`

**Review by nickolay-grechkin — APPROVED** [review]

File: frontend/src/bundles/common/components/icon-input/icon-input.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+const IconInput = ({
+    svgIcon,
+    input,
+    className = '',
+}: Properties): JSX.Element => {
+    return (
+        <div className={clsx(styles.input__wrapper, className)}>
+            {svgIcon}
```
Comment (Vitaliy-Kalenichenko) [inline frontend/src/bundles/common/components/icon-input/icon-input.tsx:18]: Check button component and follow existing naming convention

**Review by Vitaliy-Kalenichenko — APPROVED** [review]

**Pull request merged**